src/kava.js

```diff
@@ -413,6 +413,12 @@ class Kava extends BasePlugin {
     }
   }
 
+  _networkConnectionType(): string {
```

Comment:
```suggestion
  _getNetworkConnectionType(): string {
```

Reply:
I guess it would be possible but I would do it in the tidbits ticket and not now

Reply:
ok, please fix naming.
Add the change to the ticket if it isn’t there already.